tunnel.go

```diff
@@ -58,7 +58,7 @@ func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, con
 }
 
 // NewTunnels creates Tunnel clients with given options.
-func NewTunnels(account *nkn.Account, identifier string, from, to []string, tuna bool, config *Config) ([]*Tunnel, error) {
+func NewTunnels(account *nkn.Account, identifier string, from, to []string, tuna bool, config *Config, mc *nkn.MultiClient) ([]*Tunnel, error) {
```

Comment:
Should add it to comment that multiclient will be created if nil

Reply:
Yes, the comment is added.